resize.js

```diff
@@ -1,60 +1,32 @@
-'use strict';
-
-var express = require('express');
-var gm = require('gm');
-var formidable = require('formidable');
-var fs = require('fs-extra');
-var config = require('config');
-var AWS = require('aws-sdk');
-var sqlite3 = require('sqlite3').verbose();
-var responseTime = require('response-time')
-var uuid = require('node-uuid');
-var bodyParser = require('body-parser');
+"use strict";
+
+const express = require('express');
+const gm = require('gm');
+const im = gm.subClass({imageMagick: true});
```

Comment:
The resizer only used imageMagick anyway, and I presume this just loads that part.